openslides/agenda/config_variables.py

```diff
@@ -3,7 +3,6 @@
 from django.core.exceptions import ValidationError as DjangoValidationError
 from django.core.validators import MaxLengthValidator, MinValueValidator
 from django.utils.translation import ugettext as _
```

Comment:
Please remove also this line and all places where "_" as alias for ugettext is used. Add the strings also to js.